src/modules/commentListener.js

```diff
@@ -1,57 +1,119 @@
-import getComments from './APIs/comments';
-
-const buildCommentsList = (comments, commentsList) => {
-  console.log(commentsList);
-  if (comments.length > 0) {
-    commentsList.innerHTML = comments
+import getComments from "./APIs/comments";
+const involvementURL =
+  "https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/8ev0fUwJNQWCM4y1a4xa/";
+  
+  const buildCommentsList = (comments, commentsList) => {
+    if (comments.length > 0) {
+      commentsList.innerHTML = comments
       .map(
         (comment) => `
-          <li>
-            <p><em>${new Date(comment.creation_date).toLocaleDateString()} 
-            ${comment.username}</em> : ${comment.comment} </p>
-          </li>
-        `,
+        <li>
+        <p><em>${new Date(comment.creation_date).toLocaleDateString()} 
+        ${comment.username}</em> : ${comment.comment} </p>
+        </li>
+        `
       )
-      .join('');
+      .join("");
+    } else {
+      commentsList.innerHTML = "<li>No comments yet</li>";
+    }
+  };
+
+  const sendComment = async (pokemonID) => {
+    const commentForm = document.getElementById(`comment-form-${pokemonID}`);
+    const usernameInput = commentForm.querySelector(`#username-pk-${pokemonID}`);
+    const commentInput = commentForm.querySelector(`#comment-pk-${pokemonID}`);
+  const commentListComponent = document.querySelector(
+    `#comments-list-pk-${pokemonID}`
+  );
+  
+  const errorElement = document.getElementById(`error-${pokemonID}`);
+  console.log(errorElement);
+  if (usernameInput.value !== "" && commentInput.value !== "") {
+    // POST Request
+    const requestOptions = {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({
+        item_id: pokemonID,
+        username: usernameInput.value,
+        comment: commentInput.value,
+      }),
+    };
+    await fetch(
+      `${involvementURL}comments?item_id=${pokemonID}`,
+      requestOptions
+    );
```

Comment:
it would be better to move the POST REST API calls to [/src/APIs/comments.js](../../blob/development/src/modules/APIs/comments.js) to help keep your code organized and follow the DRY (Don't Repeat Yourself) principle :rocket:.
 
> This will make it easier to manage your code and reduce duplication, which is always a good thing :100: ! 

_**required changes will be something like this :eyes:**_  
At [/src/APIs/comments.js](../../blob/development/src/modules/APIs/comments.js) :heart_decoration: 
```js
export const postComment = (comment) =>{
   // place here your POST REST API call
}
```